.github/workflows/security.yaml

```
@@ -32,7 +35,60 @@ jobs:
           api-key: ${{ secrets.FOSSA_API_KEY }}
 
       - name: Run FOSSA Test
-        uses: fossas/fossa-action@main
-        with:
-          api-key: ${{ secrets.FOSSA_API_KEY }}
-          run-tests: true
+        id: fossa_test
+        env:
+          FOSSA_API_KEY: ${{ secrets.FOSSA_API_KEY }}
+        run: |
+          MAIN_BRANCH="refs/heads/main"
+          CURRENT_BRANCH="${{ github.ref }}"
+          DIFF_OPTION=""
+      
+          if [ "$CURRENT_BRANCH" != "$MAIN_BRANCH" ]; then
+            git fetch origin main
```

If we're using $MAIN_BRANCH elsewhere, maybe we should use it here too?

```suggestion
            git fetch origin $MAIN_BRANCH
```

this is absolutely a nitpick